app/Nova/Metrics/AttendancePerWeek.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Nova\Metrics;
+
+use App\Attendance;
+use Illuminate\Http\Request;
+use Laravel\Nova\Metrics\Trend;
+
+class AttendancePerWeek extends Trend
+{
+    /**
+     * Calculate the value of the metric.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return mixed
+     */
+    public function calculate(Request $request)
+    {
+        // This is slightly hacky, but it works. Otherwise, responses with a created date of midnight (as created by
+        // some forms) were pushed back to the previous day in the metric. This acts like we're in GMT while
```

Comment:
Doesn't this just cause problems for attendance created late in the day EST?

Reply:
No, I tested that. The database column has the times in EST, but the Nova metric functions assume that's GMT and want to adjust it for me, and this stops it from adjusting the time of day from the DB column.

Reply:
Interesting, ok.